[2021-08-31 19:54:35,682] {processor.py:162} INFO - Started process (PID=2484) to work on /opt/airflow/dags/test.py
[2021-08-31 19:54:35,684] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:54:35,685] {logging_mixin.py:109} INFO - [2021-08-31 19:54:35,685] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:54:35,824] {logging_mixin.py:109} INFO - [2021-08-31 19:54:35,822] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 15, in <module>
    dummy_one >> dummy_two
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 64, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1383, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1345, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(DummyOperator): test_dag_1>, <Task(DummyOperator): test_dag_2>]
[2021-08-31 19:54:35,824] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:54:35,973] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.298 seconds
[2021-08-31 19:55:06,080] {processor.py:162} INFO - Started process (PID=2499) to work on /opt/airflow/dags/test.py
[2021-08-31 19:55:06,081] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:55:06,082] {logging_mixin.py:109} INFO - [2021-08-31 19:55:06,082] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:55:06,088] {logging_mixin.py:109} INFO - [2021-08-31 19:55:06,086] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 15, in <module>
    dummy_one >> dummy_two
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 64, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1383, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1345, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(DummyOperator): test_dag_1>, <Task(DummyOperator): test_dag_2>]
[2021-08-31 19:55:06,088] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:55:06,133] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.058 seconds
[2021-08-31 19:55:36,200] {processor.py:162} INFO - Started process (PID=2523) to work on /opt/airflow/dags/test.py
[2021-08-31 19:55:36,201] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:55:36,202] {logging_mixin.py:109} INFO - [2021-08-31 19:55:36,201] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:55:36,208] {logging_mixin.py:109} INFO - [2021-08-31 19:55:36,207] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 15, in <module>
    dummy_one >> dummy_two
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 64, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1383, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1345, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(DummyOperator): test_dag_1>, <Task(DummyOperator): test_dag_2>]
[2021-08-31 19:55:36,209] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:55:36,282] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.086 seconds
[2021-08-31 19:55:42,251] {processor.py:162} INFO - Started process (PID=2524) to work on /opt/airflow/dags/test.py
[2021-08-31 19:55:42,253] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:55:42,254] {logging_mixin.py:109} INFO - [2021-08-31 19:55:42,254] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:55:42,262] {logging_mixin.py:109} INFO - [2021-08-31 19:55:42,261] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 18, in <module>
    dummy_one >> dummy_two
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 64, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1383, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1345, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(DummyOperator): test_dag_1>, <Task(DummyOperator): test_dag_2>]
[2021-08-31 19:55:42,263] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:55:42,334] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.089 seconds
[2021-08-31 19:55:52,356] {processor.py:162} INFO - Started process (PID=2532) to work on /opt/airflow/dags/test.py
[2021-08-31 19:55:52,357] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:55:52,358] {logging_mixin.py:109} INFO - [2021-08-31 19:55:52,358] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:55:52,366] {logging_mixin.py:109} INFO - [2021-08-31 19:55:52,363] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:55:52,366] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:55:52,392] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.039 seconds
[2021-08-31 19:56:22,515] {processor.py:162} INFO - Started process (PID=2547) to work on /opt/airflow/dags/test.py
[2021-08-31 19:56:22,517] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:56:22,520] {logging_mixin.py:109} INFO - [2021-08-31 19:56:22,520] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:56:22,533] {logging_mixin.py:109} INFO - [2021-08-31 19:56:22,528] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:56:22,534] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:56:22,630] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.121 seconds
[2021-08-31 19:56:52,716] {processor.py:162} INFO - Started process (PID=2570) to work on /opt/airflow/dags/test.py
[2021-08-31 19:56:52,718] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:56:52,719] {logging_mixin.py:109} INFO - [2021-08-31 19:56:52,719] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:56:52,730] {logging_mixin.py:109} INFO - [2021-08-31 19:56:52,726] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:56:52,730] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:56:52,790] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.080 seconds
[2021-08-31 19:57:22,860] {processor.py:162} INFO - Started process (PID=2585) to work on /opt/airflow/dags/test.py
[2021-08-31 19:57:22,862] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:57:22,863] {logging_mixin.py:109} INFO - [2021-08-31 19:57:22,863] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:57:22,869] {logging_mixin.py:109} INFO - [2021-08-31 19:57:22,867] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:57:22,870] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:57:22,906] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.052 seconds
[2021-08-31 19:57:53,026] {processor.py:162} INFO - Started process (PID=2600) to work on /opt/airflow/dags/test.py
[2021-08-31 19:57:53,028] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:57:53,029] {logging_mixin.py:109} INFO - [2021-08-31 19:57:53,029] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:57:53,035] {logging_mixin.py:109} INFO - [2021-08-31 19:57:53,033] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:57:53,036] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:57:53,088] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.069 seconds
[2021-08-31 19:58:23,149] {processor.py:162} INFO - Started process (PID=2623) to work on /opt/airflow/dags/test.py
[2021-08-31 19:58:23,151] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:58:23,152] {logging_mixin.py:109} INFO - [2021-08-31 19:58:23,152] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:58:23,160] {logging_mixin.py:109} INFO - [2021-08-31 19:58:23,157] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 10, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:58:23,161] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:58:23,314] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.169 seconds
[2021-08-31 19:58:37,212] {processor.py:162} INFO - Started process (PID=2632) to work on /opt/airflow/dags/test.py
[2021-08-31 19:58:37,214] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:58:37,215] {logging_mixin.py:109} INFO - [2021-08-31 19:58:37,214] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:58:37,224] {logging_mixin.py:109} INFO - [2021-08-31 19:58:37,221] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 12, in <module>
    dummy_one = DummyOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/dummy.py", line 34, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 178, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 627, in __init__
    self.dag = dag or DagContext.get_current_dag()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 733, in __setattr__
    super().__setattr__(key, value)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 784, in dag
    dag.add_task(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1604, in add_task
    raise AirflowException("Task is missing the start_date parameter")
airflow.exceptions.AirflowException: Task is missing the start_date parameter
[2021-08-31 19:58:37,225] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:58:37,307] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.100 seconds
[2021-08-31 19:58:59,304] {processor.py:162} INFO - Started process (PID=2647) to work on /opt/airflow/dags/test.py
[2021-08-31 19:58:59,306] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:58:59,307] {logging_mixin.py:109} INFO - [2021-08-31 19:58:59,307] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:58:59,394] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:59:08,496] {logging_mixin.py:109} WARNING - /home/airflow/.local/lib/python3.8/site-packages/azure/cosmos/session.py:186 SyntaxWarning: "is not" with a literal. Did you mean "!="?
[2021-08-31 19:59:08,582] {logging_mixin.py:109} WARNING - /home/airflow/.local/lib/python3.8/site-packages/azure/storage/common/_connection.py:82 SyntaxWarning: "is" with a literal. Did you mean "=="?
[2021-08-31 19:59:10,884] {logging_mixin.py:109} INFO - [2021-08-31 19:59:10,884] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 19:59:10,896] {logging_mixin.py:109} INFO - [2021-08-31 19:59:10,896] {dag.py:1852} INFO - Creating ORM DAG for joke
[2021-08-31 19:59:10,925] {logging_mixin.py:109} INFO - [2021-08-31 19:59:10,925] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 19:59:10,961] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 11.662 seconds
[2021-08-31 19:59:41,063] {processor.py:162} INFO - Started process (PID=2678) to work on /opt/airflow/dags/test.py
[2021-08-31 19:59:41,064] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 19:59:41,065] {logging_mixin.py:109} INFO - [2021-08-31 19:59:41,065] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 19:59:41,095] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 19:59:41,131] {logging_mixin.py:109} INFO - [2021-08-31 19:59:41,131] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 19:59:41,153] {logging_mixin.py:109} INFO - [2021-08-31 19:59:41,152] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 19:59:41,184] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.125 seconds
[2021-08-31 20:00:11,288] {processor.py:162} INFO - Started process (PID=2695) to work on /opt/airflow/dags/test.py
[2021-08-31 20:00:11,290] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:00:11,292] {logging_mixin.py:109} INFO - [2021-08-31 20:00:11,291] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:00:11,304] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:00:11,336] {logging_mixin.py:109} INFO - [2021-08-31 20:00:11,335] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:00:11,356] {logging_mixin.py:109} INFO - [2021-08-31 20:00:11,356] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:00:11,428] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.146 seconds
[2021-08-31 20:00:41,536] {processor.py:162} INFO - Started process (PID=2710) to work on /opt/airflow/dags/test.py
[2021-08-31 20:00:41,538] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:00:41,539] {logging_mixin.py:109} INFO - [2021-08-31 20:00:41,539] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:00:41,549] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:00:41,578] {logging_mixin.py:109} INFO - [2021-08-31 20:00:41,578] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:00:41,597] {logging_mixin.py:109} INFO - [2021-08-31 20:00:41,597] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:00:41,624] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.092 seconds
[2021-08-31 20:01:11,672] {processor.py:162} INFO - Started process (PID=2733) to work on /opt/airflow/dags/test.py
[2021-08-31 20:01:11,673] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:01:11,674] {logging_mixin.py:109} INFO - [2021-08-31 20:01:11,674] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:01:11,686] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:01:11,710] {logging_mixin.py:109} INFO - [2021-08-31 20:01:11,710] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:01:11,728] {logging_mixin.py:109} INFO - [2021-08-31 20:01:11,728] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:01:11,747] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.078 seconds
[2021-08-31 20:01:41,843] {processor.py:162} INFO - Started process (PID=2748) to work on /opt/airflow/dags/test.py
[2021-08-31 20:01:41,845] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:01:41,846] {logging_mixin.py:109} INFO - [2021-08-31 20:01:41,846] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:01:41,857] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:01:41,893] {logging_mixin.py:109} INFO - [2021-08-31 20:01:41,893] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:01:41,914] {logging_mixin.py:109} INFO - [2021-08-31 20:01:41,913] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:01:41,956] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.117 seconds
[2021-08-31 20:02:12,035] {processor.py:162} INFO - Started process (PID=2768) to work on /opt/airflow/dags/test.py
[2021-08-31 20:02:12,037] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:02:12,038] {logging_mixin.py:109} INFO - [2021-08-31 20:02:12,038] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:02:12,053] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:02:12,084] {logging_mixin.py:109} INFO - [2021-08-31 20:02:12,083] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:02:12,101] {logging_mixin.py:109} INFO - [2021-08-31 20:02:12,101] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:02:12,165] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.134 seconds
[2021-08-31 20:02:42,245] {processor.py:162} INFO - Started process (PID=2785) to work on /opt/airflow/dags/test.py
[2021-08-31 20:02:42,247] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:02:42,248] {logging_mixin.py:109} INFO - [2021-08-31 20:02:42,248] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:02:42,268] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:02:42,313] {logging_mixin.py:109} INFO - [2021-08-31 20:02:42,313] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:02:42,334] {logging_mixin.py:109} INFO - [2021-08-31 20:02:42,334] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:02:42,359] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.118 seconds
[2021-08-31 20:03:12,460] {processor.py:162} INFO - Started process (PID=2800) to work on /opt/airflow/dags/test.py
[2021-08-31 20:03:12,461] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:03:12,461] {logging_mixin.py:109} INFO - [2021-08-31 20:03:12,461] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:03:12,473] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:03:12,507] {logging_mixin.py:109} INFO - [2021-08-31 20:03:12,506] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:03:12,528] {logging_mixin.py:109} INFO - [2021-08-31 20:03:12,528] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:03:12,555] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.099 seconds
[2021-08-31 20:03:42,614] {processor.py:162} INFO - Started process (PID=2822) to work on /opt/airflow/dags/test.py
[2021-08-31 20:03:42,617] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:03:42,618] {logging_mixin.py:109} INFO - [2021-08-31 20:03:42,618] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:03:42,634] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:03:42,664] {logging_mixin.py:109} INFO - [2021-08-31 20:03:42,664] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:03:42,688] {logging_mixin.py:109} INFO - [2021-08-31 20:03:42,687] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:03:42,711] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.102 seconds
[2021-08-31 20:04:12,792] {processor.py:162} INFO - Started process (PID=2839) to work on /opt/airflow/dags/test.py
[2021-08-31 20:04:12,793] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:04:12,794] {logging_mixin.py:109} INFO - [2021-08-31 20:04:12,793] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:04:12,807] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:04:12,839] {logging_mixin.py:109} INFO - [2021-08-31 20:04:12,839] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:04:12,860] {logging_mixin.py:109} INFO - [2021-08-31 20:04:12,860] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:04:12,896] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.107 seconds
[2021-08-31 20:04:42,928] {processor.py:162} INFO - Started process (PID=2855) to work on /opt/airflow/dags/test.py
[2021-08-31 20:04:42,929] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:04:42,930] {logging_mixin.py:109} INFO - [2021-08-31 20:04:42,930] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:04:42,942] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:04:42,980] {logging_mixin.py:109} INFO - [2021-08-31 20:04:42,980] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:04:43,010] {logging_mixin.py:109} INFO - [2021-08-31 20:04:43,010] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:04:43,043] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.120 seconds
[2021-08-31 20:05:13,118] {processor.py:162} INFO - Started process (PID=2878) to work on /opt/airflow/dags/test.py
[2021-08-31 20:05:13,120] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:05:13,121] {logging_mixin.py:109} INFO - [2021-08-31 20:05:13,121] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:05:13,132] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:05:13,161] {logging_mixin.py:109} INFO - [2021-08-31 20:05:13,161] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:05:13,182] {logging_mixin.py:109} INFO - [2021-08-31 20:05:13,182] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:05:13,202] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.088 seconds
[2021-08-31 20:05:43,264] {processor.py:162} INFO - Started process (PID=2894) to work on /opt/airflow/dags/test.py
[2021-08-31 20:05:43,265] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:05:43,266] {logging_mixin.py:109} INFO - [2021-08-31 20:05:43,266] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:05:43,278] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:05:43,310] {logging_mixin.py:109} INFO - [2021-08-31 20:05:43,310] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:05:43,332] {logging_mixin.py:109} INFO - [2021-08-31 20:05:43,331] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:05:43,358] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.101 seconds
[2021-08-31 20:06:13,413] {processor.py:162} INFO - Started process (PID=2919) to work on /opt/airflow/dags/test.py
[2021-08-31 20:06:13,414] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:06:13,415] {logging_mixin.py:109} INFO - [2021-08-31 20:06:13,415] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:06:13,429] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:06:13,451] {logging_mixin.py:109} INFO - [2021-08-31 20:06:13,451] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:06:13,469] {logging_mixin.py:109} INFO - [2021-08-31 20:06:13,468] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:06:13,496] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.088 seconds
[2021-08-31 20:06:43,585] {processor.py:162} INFO - Started process (PID=2934) to work on /opt/airflow/dags/test.py
[2021-08-31 20:06:43,586] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:06:43,587] {logging_mixin.py:109} INFO - [2021-08-31 20:06:43,587] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:06:43,598] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:06:43,629] {logging_mixin.py:109} INFO - [2021-08-31 20:06:43,629] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:06:43,655] {logging_mixin.py:109} INFO - [2021-08-31 20:06:43,655] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:06:43,679] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.101 seconds
[2021-08-31 20:07:13,736] {processor.py:162} INFO - Started process (PID=2949) to work on /opt/airflow/dags/test.py
[2021-08-31 20:07:13,737] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:07:13,738] {logging_mixin.py:109} INFO - [2021-08-31 20:07:13,738] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:07:13,748] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:07:13,783] {logging_mixin.py:109} INFO - [2021-08-31 20:07:13,782] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:07:13,803] {logging_mixin.py:109} INFO - [2021-08-31 20:07:13,803] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:07:13,830] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.098 seconds
[2021-08-31 20:07:43,894] {processor.py:162} INFO - Started process (PID=2972) to work on /opt/airflow/dags/test.py
[2021-08-31 20:07:43,895] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:07:43,896] {logging_mixin.py:109} INFO - [2021-08-31 20:07:43,896] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:07:43,913] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:07:43,942] {logging_mixin.py:109} INFO - [2021-08-31 20:07:43,942] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:07:43,962] {logging_mixin.py:109} INFO - [2021-08-31 20:07:43,961] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:07:43,985] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.094 seconds
[2021-08-31 20:08:14,025] {processor.py:162} INFO - Started process (PID=2987) to work on /opt/airflow/dags/test.py
[2021-08-31 20:08:14,026] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:08:14,027] {logging_mixin.py:109} INFO - [2021-08-31 20:08:14,027] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:08:14,039] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:08:14,069] {logging_mixin.py:109} INFO - [2021-08-31 20:08:14,068] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:08:14,084] {logging_mixin.py:109} INFO - [2021-08-31 20:08:14,084] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:08:14,110] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.090 seconds
[2021-08-31 20:08:44,197] {processor.py:162} INFO - Started process (PID=3002) to work on /opt/airflow/dags/test.py
[2021-08-31 20:08:44,199] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:08:44,200] {logging_mixin.py:109} INFO - [2021-08-31 20:08:44,199] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:08:44,213] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:08:44,239] {logging_mixin.py:109} INFO - [2021-08-31 20:08:44,239] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:08:44,257] {logging_mixin.py:109} INFO - [2021-08-31 20:08:44,257] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:08:44,281] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.087 seconds
[2021-08-31 20:09:14,345] {processor.py:162} INFO - Started process (PID=3026) to work on /opt/airflow/dags/test.py
[2021-08-31 20:09:14,347] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:09:14,348] {logging_mixin.py:109} INFO - [2021-08-31 20:09:14,348] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:09:14,358] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:09:14,389] {logging_mixin.py:109} INFO - [2021-08-31 20:09:14,389] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:09:14,413] {logging_mixin.py:109} INFO - [2021-08-31 20:09:14,413] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:09:14,457] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.115 seconds
[2021-08-31 20:09:40,537] {processor.py:162} INFO - Started process (PID=3042) to work on /opt/airflow/dags/test.py
[2021-08-31 20:09:40,538] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:09:40,538] {logging_mixin.py:109} INFO - [2021-08-31 20:09:40,538] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:09:40,550] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:09:40,586] {logging_mixin.py:109} INFO - [2021-08-31 20:09:40,585] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:09:40,605] {logging_mixin.py:109} INFO - [2021-08-31 20:09:40,604] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:09:40,639] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.108 seconds
[2021-08-31 20:10:10,683] {processor.py:162} INFO - Started process (PID=3057) to work on /opt/airflow/dags/test.py
[2021-08-31 20:10:10,684] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:10:10,685] {logging_mixin.py:109} INFO - [2021-08-31 20:10:10,685] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:10:10,696] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:10:10,727] {logging_mixin.py:109} INFO - [2021-08-31 20:10:10,726] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:10:10,747] {logging_mixin.py:109} INFO - [2021-08-31 20:10:10,747] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:10:10,766] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.090 seconds
[2021-08-31 20:10:40,847] {processor.py:162} INFO - Started process (PID=3081) to work on /opt/airflow/dags/test.py
[2021-08-31 20:10:40,849] {processor.py:613} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2021-08-31 20:10:40,850] {logging_mixin.py:109} INFO - [2021-08-31 20:10:40,850] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2021-08-31 20:10:40,867] {processor.py:623} INFO - DAG(s) dict_keys(['joke']) retrieved from /opt/airflow/dags/test.py
[2021-08-31 20:10:40,899] {logging_mixin.py:109} INFO - [2021-08-31 20:10:40,898] {dag.py:1833} INFO - Sync 1 DAGs
[2021-08-31 20:10:40,926] {logging_mixin.py:109} INFO - [2021-08-31 20:10:40,925] {dag.py:2306} INFO - Setting next_dagrun for joke to 2021-08-29 00:00:00+00:00
[2021-08-31 20:10:40,957] {processor.py:170} INFO - Processing /opt/airflow/dags/test.py took 0.114 seconds
